website/docs/language/v1-compatibility-promises.mdx

```diff
@@ -82,7 +82,7 @@ providers) will retain their current functionality:
   [`locals`](/docs/language/values/locals#declaring-a-local-value) blocks
   for declaring the various kinds of named values in a module.
 * [`terraform`](/docs/language/settings) blocks, including the nested
-  [`required_version`](/docs/language/settings#specifying-a-required-opentf-version)
+  [`required_version`](/docs/language/settings#specifying-a-required-tofu-version)
```

Comment:
This should be `opentofu` to match simplified heading:
https://github.com/julienlevasseur/opentofu/blob/9624f747fea71e0226b0216edb2b4a2b76794115/website/docs/language/settings/index.mdx#L39